src/containers/_RegFormContainer.jsx

```diff
@@ -53,33 +56,37 @@ const RegFormContainer = () => {
 						Just a few more steps and you`re done! We hate paperwork, too.
 					</RegForm.SubTitle>
 
-					<RegForm.Frame onClick={ () => setEmailFocus(true) } >
-						<RegForm.Label
-							focus={ emailFocus ? true : false }
-						>
-							Email
-						</RegForm.Label>
+					<RegForm.Frame>
 						<RegForm.Input
 							type="text"
+							id="email"
+							placeholder=" "
 							value={ email }
 							onChange={ ({ target }) => setEmail(target.value) }
 							error={ emailError ? true : false }
 						/>
-						{ emailError && <p>Email is required!</p> }
-					</RegForm.Frame>
-
-					<RegForm.Frame onClick={ () => setPasswordFocus(true) } >
 						<RegForm.Label
-							focus={ passwordFocus ? true : false }
+							htmlFor="email"
 						>
```

Comment:
I was wondering, what is your thought on the text for tags likes these?

```html
<!-- option 1 -->
<button
  title="click me"
>
  the click is real
</button>

<!-- option 2 -->
<button
  title="click me"
>the click is real
</button>

<!-- option 3 -->
<button
  title="click me"
>the click is real</button>
```

Reply:
I personally like option 1.